Occasionally the cp -p fails with a non-zero return code.  This will cause
the system abort the build.

Upstream-Status: Submitted [RPM5 maintainer]

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

Index: rpm/rpmdb/Makefile.am
===================================================================
--- rpm.orig/rpmdb/Makefile.am
+++ rpm/rpmdb/Makefile.am
@@ -234,11 +234,14 @@ lcov-report:
 #lcov-upload: lcov
 #	rsync -rvz -e ssh --delete lcov/* ???
 
+$(builddir)/logio_recover_template: $(srcdir)/logio_recover_template
+	@if test ".$(builddir)" != ".$(srcdir)"; then \
+		cp -fp $(srcdir)/logio_recover_template \
+		      $(builddir)/logio_recover_template ; \
+	 fi
+
 logio_BUILT = logio_auto.c logio_autop.c logio_auto.h logio_template
-$(logio_BUILT):	logio.awk logio.src logio_recover_template
-	@test -e $(builddir)/logio_recover_template || \
-		cp -p $(srcdir)/logio_recover_template \
-		      $(builddir)/logio_recover_template
+$(logio_BUILT):	logio.awk logio.src $(builddir)/logio_recover_template
 	@rm -f $(logio_BUILT)
 	@$(AWK) -f $(srcdir)/logio.awk \
 		-v header_file=logio_auto.h \
